ansible/library/announce_routes.py

```diff
@@ -465,24 +470,41 @@ def fib_t0(topo, ptf_ip, no_default_route=False, action="announce"):
     spine_asn = common_config.get("spine_asn", SPINE_ASN)
     leaf_asn_start = common_config.get("leaf_asn_start", LEAF_ASN_START)
     tor_asn_start = common_config.get("tor_asn_start", TOR_ASN_START)
+    ipv6_address_pattern = common_config.get("ipv6_address_pattern", IPV6_ADDRESS_PATTERN_DEFAULT_VALUE)
+    enable_ipv6_routes_generation = common_config.get("enable_ipv6_routes_generation",
```

Comment:
Wenda, do you mind to explain more on the relationship between this change and the topology, I somehow cannot connect the dot between these 2.

![image](https://github.com/user-attachments/assets/3b232d61-5837-4a56-aecc-c4efd2f9ffef)

Reply:
Yes, those parameters are added to control the routes generation.

For ipv6 only topology, we can set enable_ipv4_routes_generation to false, then the code will not try to generate and advertise ipv4 routes.

And if want to mock aggregate address on peer, we can add key `aggregate_prefixes` and a list in its config, then the code will advertise the aggregate address.

I raise a PR as example: https://github.com/sonic-net/sonic-mgmt/pull/16213

Reply:
I see. Do you mind to update the new PR and having all related topology updated?

Also in this case, should we rename the topology to be `topo_t*-isolated-d*u*s*-v6` to be more explicit?